### PR TITLE
Hook to fix weirdness with splash page bg

### PR DESCRIPTION
**Pull request by sfowles**

### CODE REVIEW

File: src/Components/Presentational/Splash.js [hunk 1]
```diff
@@ -26,9 +26,10 @@ export default function Splash() {
   const classes = useStyles();
   const theme = useTheme();
 
-  theme.palette.type === 'light'
-    ? (document.body.style.backgroundColor = color.c50)
-    : (document.body.style.backgroundColor = color.g900);
+  useEffect(() => {
+    document.body.className = 'splash';
+    return () => { document.body.className = ''; }
```
Comment (Jake-Lindvall) [inline src/Components/Presentational/Splash.js:31]: This is super clever and clean ❤️

**Review by Jake-Lindvall — DISMISSED** [review]
👍

**Review by ghost — APPROVED** [review]
looks good to me! Good catch making sure that the splash changes depending on light/dark theme 👍

File: src/Components/Presentational/Splash.js (hunk 1 above)
Comment (sfowles) [reply src/Components/Presentational/Splash.js:31]: Thanks dude!!

**Review by Jake-Lindvall — APPROVED** [review]

**Pull request merged**